<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TodoEditForm</class>
 <widget class="QWidget" name="TodoEditForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Todo</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QToolButton" name="ui_tbState">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="task-complete">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="ui_tbPRI0">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="flag-red">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">PRI</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="ui_tbPRI1">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="flag-yellow">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">PRI</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="ui_tbPRI2">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="flag-green">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">PRI</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ui_leLabel"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="ui_tbDelete">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="delete">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="ui_tbSave">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="document-save">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="ui_teContent">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ui_tbState</tabstop>
  <tabstop>ui_tbPRI0</tabstop>
  <tabstop>ui_tbPRI1</tabstop>
  <tabstop>ui_tbPRI2</tabstop>
  <tabstop>ui_tbDelete</tabstop>
  <tabstop>ui_tbSave</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <signal>completed(bool)</signal>
  <slot>changedPRI(int,bool)</slot>
 </slots>
 <buttongroups>
  <buttongroup name="PRI"/>
 </buttongroups>
</ui>
